packages/payload/src/queues/operations/rest/run.ts

```diff
@@ -0,0 +1,77 @@
+import type { Endpoint } from '../../../config/types.js'
```

Comment:
wherever / however you name this file, it should have the word `endpoint` involved

Reply:
prob `queues/endpoint.ts` - done